examples/text_to_image/train_text_to_image_lora.py

```diff
@@ -415,7 +416,18 @@ def main():
     unet = UNet2DConditionModel.from_pretrained(
         args.pretrained_model_name_or_path, subfolder="unet", revision=args.revision
     )
-
+    unet.requires_grad_(False)
+    vae.requires_grad_(False)
+    
+    params = itertools.chain(
```

Comment:
Why not just to `text_encoder.requires_grad_(False)` ?

Reply:
+1,  , this way of freezing is only needed for the textual inversion
here we can use `text_encoder.requires_grad_(False)`